scheduler/scheduler.go

```diff
@@ -8,6 +8,36 @@ import (
 	"github.com/pganalyze/collector/util"
 )
 
+type Scheduler struct {
+	TenSecond Group
+	OneMinute Group
+	TenMinute Group
+}
+
+func GetScheduler() (scheduler Scheduler, err error) {
```

Comment:
Personally I found `GetSchedulerGroups` to be more descriptive, since the structs being returned are also called `Group`, so I'd be inclined to keep the old name with the new structure.

I feel a "scheduler" might be more something that comes up with a schedule given user input (and returns that schedule) which is not what we're doing here.

Reply:
If we must have a two-word name for this struct, the best option might be `TaskScheduler`. But since there are no other schedule-like structs this might conflict with, I think it's fine to use `Scheduler`.

Reply:
Eh, I think `Scheduler` is fine if you prefer - we can leave as-is.